site-src/geps/gep-851.md

```diff
@@ -0,0 +1,80 @@
+# GEP-851: Allow Multiple Certificate Refs per Gateway Listener
+
+* Issue: [#851](https://github.com/kubernetes-sigs/gateway-api/issues/851)
+* Status: Implementable
+
+## TLDR
+
+Replace `CertificateRef` field with a `CertificateRefs` field in Gateway
+Listeners.
+
+## Goals
+
+* Enable specifying RSA and ECDSA certs on the same Listener.
+* Provide a path to future expansion that could include:
+  * Support for referencing certificates for different hostnames (maybe as part
+    of a self-service TLS approach).
+  * Support for including new and old certificates as part of renewal process.
+
```

Comment:
* Certificate pining: enable implementations that require certain certificates for legacy clients while exposing a "normal" certificate to non-legacy clients.

Reply:
Added, thanks!